templates/conf.py.jj2

```diff
@@ -7,50 +7,48 @@ DESCRIPTION = (
 {% endfor %}
     ''
 )
-extensions = [
-    'sphinx.ext.autodoc',
-    'sphinx.ext.doctest',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.viewcode',
-{%block SPHINX_EXTENSIONS%}
-{%endblock%}
-]
-{%block custom_varaibles%}
-{%endblock%}
-intersphinx_mapping = {
```

Comment:
this is the only difference between the two conf.py.jj2 , and it doesnt seem worthwhile having two different templates if this is the only difference.